public/js/io/model.js

```diff
@@ -140,13 +140,21 @@ angular.module('app').service('ioModel', function (connection, $q) {
     this.importBundle = function (bundle, datasourceRef) {
         const additions = [];
         const messages = [];
+        const replace = [];
 
         const layerPromises = [];
         for (const layer of bundle.layers) {
             const p = getLayer(layer._id).then(l => {
                 if (l) {
-                    messages.push('Layer was not imported because it already exists in database: ' + l.name);
-                    return;
+                    if (layer.replace === true) {
+                        messages.push('This Layer was replace: ' + l.name);
```

Comment:
Wording:
* `Layer` -> `layer`
* `replace` -> `replaced`

Also, strings need to be translatable (see usage of `gettextCatalog`)